eslint.config.js

```diff
@@ -0,0 +1,44 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
+import neostandard from 'neostandard';
+import nodePlugin from 'eslint-plugin-n';
+import pluginPromise from 'eslint-plugin-promise';
+import importPlugin from 'eslint-plugin-import';
+
+export default [
+  pluginJs.configs.recommended,
+  ...neostandard(),
+  nodePlugin.configs['flat/recommended'],
+  pluginPromise.configs['flat/recommended'],
+  importPlugin.flatConfigs.recommended,
+  {
+    ignores: [
```

Comment:
I've learned that global ignores have to be alone in the first config element

Reply:
I'm not sure I understand, mind to elaborate?

Reply:
https://eslint.org/docs/latest/use/configure/ignore#ignoring-files

Doesn't have to be the first element in the config array but must have no other keys.